tests/test_proxy_functional.py

```diff
@@ -241,7 +241,6 @@ async def test_https_proxy_unsupported_tls_in_tls(
     await asyncio.sleep(0.1)
 
 
-@pytest.mark.usefixtures("uvloop_loop")
```

Comment:
This syntax is correct and is used when the fixture value isn't necessary within the test function.

Reply:
Yes, but that leads to running of test with the wrong event loop